examples/aws/storage-efs/README.md

```diff
@@ -0,0 +1,242 @@
+# Deploying Kubeflow with AWS EFS as Persistent Storage
```

Comment:
This readme can be cut down. Can we point the non EFS specific steps to existing README. The reason is things might get outdated and you don't want to keep updating same steps at multiple places.

Reply:
Keep this readme only specific to EFS related instruction, remaining steps just provide link + steps from existing README.

Reply:
So not include the kustomization deployment option at all and instead just the post-deployment kubectl install command as - 
`kubectl apply -k "github.com/kubernetes-sigs/aws-efs-csi-driver/deploy/kubernetes/overlays/stable/?ref=release-1.3"`

This is the only part that is really different. Based on the above comments this README can basically be trimmed to the following - 

1. Link to base kubeflow deployement and other pre-reqs 
2. Create the necessary IAM Role
3. Single line driver installation specified above (note that we will not be able to configure the region for the driver image, so I need to check this once)
4. The creation of the EFS file system is almost exactly the same as this document - https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html. If we do not want to duplicate as per Gautam's comment, maybe we can point them to this link instead ? 
5. We keep sections 6, 7, 8 as is.

Reply:
use tag.